### ISSUE TITLE
Aspose PHP

### ISSUE THREAD

**Issue by dev-archana-soni** [aspose-imaging/Aspose.Imaging-for-Java#16]
My main 3 steps/tasks are like

1. Load .tiff image in m chrome browser with zoom in zoom out and also can rotate it
2. Get coordinates of all the texts from image
3. Highlight specific text from particular image using the coordinates

These all 3 above task can do via aspose any demo for it?

Do i need to purchase aspose for above 3 stuffs? Please make sure i want it only in PHP / LARAVEL.

**Comment by DenysSitkoA** [comment]
Hello, @Archana-1993 ,
Aspose.Imaging library is available for .NET/Java platforms now. As concerns the license, you get watermarked output images and can not work with pixel data in evaluation mode.
Please, look through the link: https://products.aspose.com/imaging/

**Comment by dev-archana-soni** [comment]
Yes i know. @DenisSitko 

But, What is this?
https://github.com/aspose-imaging/Aspose.Imaging-for-Java/tree/master/Plugins/Aspose_Imaging_Java_for_PHP

It will help to me?

**Comment by ESidenko** [comment]
@Archana-1993 
> 
> But, What is this? https://github.com/aspose-imaging/Aspose.Imaging-for-Java/tree/master/Plugins/Aspose_Imaging_Java_for_PHP
> 
> It will help to me?
Yes, this can help but you need to have a Tomcat server with Java Bridge module. The Java Bridge is a middle tier between your PHP code and Aspose.Imaging for Java API.
